src/tls.c

```diff
@@ -14278,7 +14422,19 @@ int TLSX_Parse(WOLFSSL* ssl, const byte* input, word16 length, byte msgType,
                 ret = EC_PARSE(ssl, input + offset, size, isRequest,
                         &ssl->extensions);
                 break;
-
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+            case TLSX_CKS:
+                WOLFSSL_MSG("CKS extension received");
+                if (!IsAtLeastTLSv1_3(ssl->version) ||
+                    (msgType != client_hello &&
+                    msgType != encrypted_extensions)) {
```

Comment:
add a space at the front to have msgTypes aligned.

Reply:
Fixed in next commit.